okonomiyaki/file_formats/_package_info.py

```diff
@@ -176,6 +183,18 @@ def from_string(cls, s):
                     if value != 'UNKNOWN':
                         kw[attr_name] = value
 
+        if metadata_version_info >= (2, 1):
+            msg_body = msg.get_payload()
+            if msg_body is not None and msg_body != '':
+                if 'description' in kw:
+                    raise ValueError(
```

Comment:
I am not sure that we should raise an error here. Probably provide a warning. Let have an offline chat when time permits